app/views/pages/_badges.html.erb

```diff
@@ -0,0 +1,30 @@
+<h2 class="govuk-heading-m">
+  Your 2020 to 2021 trainees
+</h2>
+
+<div class="govuk-grid-row govuk-!-margin-bottom-6">
+  <div class="govuk-grid-column-one-third">
+    <%= render TraineeStatusCard::View.new(state: :draft, target: trainees_path("state[]": "draft"), user: current_user) %>
+  </div>
+  <div class="govuk-grid-column-one-third">
+    <%= render TraineeStatusCard::View.new(state: :submitted_for_trn, target: trainees_path("state[]": "submitted_for_trn"), user: current_user) %>
+  </div>
+  <div class="govuk-grid-column-one-third">
+    <%= render TraineeStatusCard::View.new(state: :trn_received, target: trainees_path("state[]": "trn_received"),user: current_user) %>
+  </div>
+</div>
+
+<div class="govuk-grid-row">
```

Comment:
The prototype has a bigger bottom margin on this row too.

Reply:
Thanks, fixed